pyuploadtool/changelog/changelog_spec.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from .changelog import Changelog
+from .commit import ChangelogCommit
+
+
+class ConventionalCommitChangelog(Changelog):
+    @property
+    def structure(self) -> dict:
```

Comment:
This should be a static or class method, not a property. It's not object-specific.

Reply:
Agreed, yes.